skia/goldens/src/goldens_gpu.cpp

```diff
@@ -0,0 +1,102 @@
+#ifdef TESTING
+
+// Don't compile this file as part of the "tests" project.
+
+#else
+
+#include "goldens_grid.hpp"
+
+#include "GLFW/glfw3.h"
+#include "skia_renderer.hpp"
+#include "skia/include/core/SkSurface.h"
+#include "skia/include/gpu/GrDirectContext.h"
+#include "skia/include/gpu/gl/GrGLInterface.h"
+#include "skia/include/gpu/gl/GrGLAssembleInterface.h"
+#include "skia/third_party/externals/libpng/png.h"
+#include "tools/write_png_file.hpp"
```

Comment:
I don't see where this file got imported. Do we need to add the tools directory?

Reply:
didn't yet. I only ported enough to test 1 app (the cpu one). Others to follow.